src/scripts/tag_tracking_plus.css

```diff
@@ -0,0 +1,29 @@
+#tag-tracking-plus ul::after {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 36px;
+  padding: 5px 10px;
+  border-radius: 3px;
+  margin-top: 10px;
+
+  background-color: rgba(var(--white-on-dark), 0.13);
+  color: rgba(var(--white-on-dark), 0.6);
+  font-size: .875rem;
+}
```

Comment:
If we felt like it, we could bikeshed this, but I really don't think it's at all important :D

```suggestion
#tag-tracking-plus ul::after {
  display: flex;
  justify-content: center;
  align-items: center;
  min-height: 2rem;
  padding: 5px 10px;
  border-radius: 3px;
  margin-top: 10px 5px;

  background-color: rgba(var(--white-on-dark), 0.13);
  color: rgba(var(--white-on-dark), 0.6);
  font-size: 1rem;
}
```

Reply:
additional option:
```css
border: 1px dashed rgba(var(--white-on-dark), 0.13);
```

<img width="350" src="https://user-images.githubusercontent.com/8336245/219976787-c262bb29-9d2b-4215-98ad-a1c1c8d89571.png">